tools/actions/commit-queue.sh

```diff
@@ -19,7 +19,7 @@ commit_queue_failed() {
 
   # shellcheck disable=SC2154
   cqurl="${GITHUB_SERVER_URL}/${OWNER}/${REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
-  body="<details><summary>Commit Queue failed</summary><pre>$(cat output)</pre><a href='$cqurl'>$cqurl</a></details>"
+  body="<details><summary>Commit Queue failed</summary><pre>$(sed -e 's/</\&lt;/g' -e 's/>/\&gt;/g' output)</pre><a href='$cqurl'>$cqurl</a></details>"
```

Comment:
This should probably also escape `&` as `&amp;`:
```suggestion
  body="<details><summary>Commit Queue failed</summary><pre>$(sed -e 's/&/\&amp;/g' -e 's/</\&lt;/g' -e 's/>/\&gt;/g' output)</pre><a href='$cqurl'>$cqurl</a></details>"
```

Reply:
There's nothing wrong with `&` inside of a pre tag AFAIK, so I don't think it needs to be escaped.

Reply:
Of course there is, e.g. if you use ``meta: replace < with &lt; in `commit-queue` comment`` as the commit message, it would be displayed as:

<pre>
meta: replace &lt; with &lt; in `commit-queue` comment
</pre>

Reply:
Oh, that didn't occur to me.